test-suite/hestonmodel.cpp

```diff
@@ -1440,7 +1440,7 @@ void HestonModelTest::testExpansionOnAlanLewisReference() {
 }
 
 void HestonModelTest::testExpansionOnFordeReference() {
-    BOOST_TEST_MESSAGE("Testing expansion on Forde reference prices...");
+    BOOST_TEST_MESSAGE("Testing expansion on Alan Lewis reference prices...");
```

Comment:
Hi Luigi,

I have changed the file in pull request #48 (Heston SLV model, part 2), since I cannot easily changed this one.

Kind regards,
Johannes